### PR TITLE
chore: removed revert section from changelog

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description

Removed the revert PR which surfaced up in the recent CHANGELOG.

## Related issue(s)

-

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_

    1. Go here
    2. Do this

-   [ ] Did it pass in Desktop?
-   [ ] Did it pass in Mobile?
-   [ ] Did it pass in iPad?

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ ] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ ] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: packages/slider/CHANGELOG.md [hunk 1]
```diff
@@ -11,10 +11,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 -   **slider:** allow handler to be focusable when slider is editable ([23f8b89](https://github.com/adobe/spectrum-web-components/commit/23f8b8908a288f246f6330292b7ff090f931822a))
```
Comment (Rocss) [inline packages/slider/CHANGELOG.md:11]: Also remove this if you remove the revert

Comment (Rajdeepc) [reply packages/slider/CHANGELOG.md:11]: The revert section has been removed. May I know why do you want me to remove this line?

Comment (Rajdeepc) [reply packages/slider/CHANGELOG.md:11]: My bad! I overlooked

**Review by rubencarvalho — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,10 +20,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 -   **slider:** allow handler to be focusable when slider is editable ([23f8b89](https://github.com/adobe/spectrum-web-components/commit/23f8b8908a288f246f6330292b7ff090f931822a))
```
Comment (Rocss) [inline CHANGELOG.md:20]: This one too I think

**Review by Rocss — APPROVED** [review]

**Pull request merged**